src/Factories/FunFunFactory.php

```diff
@@ -1372,4 +1373,11 @@ public function setContentProvider( ContentProvider $contentProvider ): void {
 	private function getContentProvider(): ContentProvider {
 		return $this->pimple['content_provider'];
 	}
+
+	public function newMailTemplateFilenameTraversable(): \Traversable {
```

Comment:
- I'd prefer being specific in return types (and generous in parameter type hints) unless there is a compelling reason against it
- I reckon we agreed on using `use` statements instead of inline absolute namespaces